scalableminds/webknossos
# Remove Config Defaults from Scala code

### Steps to test:
- remove a required config value from application.conf
- wK startup should fail with readable exception
- with the application.conf in the repo, it should run
- also test that [standalone datastores](https://github.com/scalableminds/webknossos/wiki/Set-up-a-standalone-datastore-locally) still work

### TODO/Decisions
- [x] also require config keys used by frameworks, not used by us? → No, we decided that’s redundant and doesn’t help, as we cannot make guarantees about its completeless
- [x] figure out what changes are necessary in kube/migration
- [x] decide wether to name the nested objects all-lower-case to match config keys → No, keep it as it is

### Issues:
- fixes #2326 

------
